git-hooks/no-commit-to-default-branch

```diff
@@ -0,0 +1,18 @@
+#! /usr/bin/env bash
+# source https://github.com/bettermarks/.github/git-hooks/
+echo "executing $0 in $(pwd)"
+set -eu
+current_branch=$(git rev-parse --abbrev-ref HEAD)
```

Comment:
When somebody follow the suggestion to configure the global template, and run git init in a repo, nothing will happen, until a `.pre-commit-config.yaml` exists,
in which case you get a friendly reminder to install `pre-commit`.

I think it really is an edge case (but I will try to make it more clear from the readme), that somebody would want to commit the `.pre-commit-config.yaml` file as the first thing in the repository, and configure it to prevent commits to the default branch...

But I like the much more readable command you suggest. Thx for making me aware.
And I will even add an exit 0 for the case of no commits existing, just in case.
For everything else, ... let's see if the problem ever occurs.